pkg/term/term_windows.go

```diff
@@ -1,12 +1,15 @@
 // +build windows
+
 package term
 
 import (
+	"fmt"
 	"io"
 	"os"
+	"os/signal"
 
-	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/pkg/term/winconsole"
+	// "github.com/Sirupsen/logrus"
```

Comment:
@RickWieman Are you suggesting that we use logrus to generate the returned errors (vs. only when writing to the log file)? From my perusal of logrus it does not support that. However, I'm happy to change to whatever is best for Docker.

@jhowardmsft Easily changed. I thought when I scanned the Docker code I failed to see a consistent pattern.